modules/ossm-control-plane-profiles.adoc

```diff
@@ -12,7 +12,7 @@ When you configure control plane profiles, which follow the same syntax as the `
 [id="ossm-create-configmap_{context}"]
 == Creating the ConfigMap
 
-To add custom profiles, you must first create a ConfigMap named `smcp-templates` in the `openshift-operators` project and then mount the ConfigMap in the Operator container at: `/usr/local/share/istio-operator/templates`.
+To add custom profiles, you must create a ConfigMap named `smcp-templates` in the `openshift-operators` project. The Operator container automatically mounts the ConfigMap.
```

Comment:
```suggestion
To add custom profiles, you must create a `ConfigMap` named `smcp-templates` in the `openshift-operators` project. The Operator container automatically mounts the `ConfigMap`.
```

Reply:
Thanks for pointing this out, Rolfe. I've made this edit.